src/Symfony/Component/DependencyInjection/Tests/Fixtures/config/packages/ping.yaml

```diff
@@ -0,0 +1,6 @@
+imports:
+    - { resource: './third_a.yaml' }
+    - { resource: './third_b.yaml' }
+
+third:
+    foo: ping
```

Comment:
They will be loaded in top-down order using the prepend method, meaning that the first one imported will have priority over the rest. This matters when there are collisions between them (like the test case shows).

We can think about it as `array_merge()` works (per node level).

Reply:
Nope, it's not the same order as the regular imports because appending configs will output in the opposite order than prepend (before merging process). But it makes sense to me as it will be the expected behavior according to the `prependExtension()` method. Here, you want to prepend everything that you import, and that will include caring about sorting the recursive imports when necessary.

Note that, until now, using imports in the prepend method was not possible or defective (due to the reason given in the PR's description), so this is a new scenario, convention and a behavior change for the `ContainerConfiguration::import()` method (when used in the `prependExtension()` only) that must be well documented.

Reply:
Is there a way to keep the regular order within the same yaml file even for recursive imports?
I feel like that might make more sense (the fact this is prepended is external to the file so within the file I would expect to not know about this ordering concern - or at least to easily forget about it - aka FTW pending :) )

Reply:
I made the changes to keep the regular order within the same file, while the resulting extension configs are prepended at the end.